### PR TITLE
add CloudTx Chain

### PR DESCRIPTION
**Pull request by massanto**
add CloudTX mainnet adn CLoudTx Testnet

### CODE REVIEW

**Review by Ukandu147 — APPROVED** [review]

**Review by ligi — CHANGES_REQUESTED** [review]
short name must be unique - but found `CLD` more than once

**Review by ligi — CHANGES_REQUESTED** [review]
tl;dr: it is now required to put the binary icon into iconDownload

why: it was a PITA to always `ipfs get` in the review process - sometimes it takes *really long* or even fails (especially with pinata) - this took significant time (which adds up with the amount of chains added) and slowed down the review process. Also sometimes CIDs become unavailable after - big mess. Hence it is now required to put them in iconDownload (still named via CID)

**Comment by massanto** [comment]
> tl;dr: it is now required to put the binary icon into iconDownload
> 
> why: it was a PITA to always `ipfs get` in the review process - sometimes it takes _really long_ or even fails (especially with pinata) - this took significant time (which adds up with the amount of chains added) and slowed down the review process. Also sometimes CIDs become unavailable after - big mess. Hence it is now required to put them in iconDownload (still named via CID)

thanks you, I don't know add CID on iconDownload.
just code name ? like QmSEsi71AdA5HYH6VNC5QUQezFg1C7BiVQJdx1VVfGz3g3

**Comment by ligi** [comment]
size of the icon is wrong:

QmSEsi71AdA5HYH6VNC5QUQezFg1C7BiVQJdx1VVfGz3g3 PNG 713x830 713x830+0+0 8-bit sRGB 121957B 0.000u 0:00.000

**Comment by massanto** [comment]
> 

noted, we will change

File: _data/icons/cloudtx.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "ipfs://QmSEsi71AdA5HYH6VNC5QUQezFg1C7BiVQJdx1VVfGz3g3",
+    "width": 713,
+    "height": 830,
+    "format": "PNG"
```
Comment (ligi) [inline _data/icons/cloudtx.json:6]:
```suggestion
    "format": "png"
```

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/chains/eip155-31223.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "CloudTx Mainnet",
+  "chain": "CLD",
+  "icon": "cloudtx",
+  "rpc": ["https://mainnetrpc.num.network"],
```
Comment (massanto) [inline _data/chains/eip155-31223.json:5]: i will change the RPC mainnet to https://mainnet-rpc.cloudtx.finance
